<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_statistics</class>
 <widget class="QDialog" name="Dialog_statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tbWidget">
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::TextElideMode::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_year">
      <attribute name="title">
       <string>Статистики за год</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_month">
      <attribute name="title">
       <string>Статистики за месяц</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="cB_month_numer">
         <property name="currentText">
          <string>Январь</string>
         </property>
         <property name="maxVisibleItems">
          <number>12</number>
         </property>
         <item>
          <property name="text">
           <string>Январь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Февраль</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Март</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Апрель</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Май</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Июнь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Июль</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Август</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Сентябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Октябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ноябрь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Декабрь</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
